File: app/src/main/java/com/example/sherryuan/launchnote/models/PicNote.java
package com.example.sherryuan.launchnote.models;

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.Ignore;
import android.arch.persistence.room.PrimaryKey;
import android.graphics.Bitmap;

/**
 * Model class for representing one photo
 */
@Entity
public class PicNote {

    @PrimaryKey
    private String id;
    private String imageUri;
    private String description;
    @Ignore
    private Bitmap image;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getImageUri() {
        return imageUri;
    }

    public void setImageUri(String imageUri) {
        this.imageUri = imageUri;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public Bitmap getImage() {
        return image;
    }

    public void setImage(Bitmap image) {
        this.image = image;
    }
}

File: app/src/main/java/com/example/sherryuan/launchnote/models/PicNote.kt
package com.example.sherryuan.launchnote.models

import android.arch.persistence.room.Entity
import android.arch.persistence.room.Ignore
import android.arch.persistence.room.PrimaryKey
import android.graphics.Bitmap

/**
 * Model class for representing one photo
 */
@Entity
class PicNote (
    @PrimaryKey
    var id: String,
    var imageUri: String,
    var description: String,
    @Ignore
    var image: Bitmap? = null
)
